### PR TITLE
Unused parts of the old notification system are removed

### PR DESCRIPTION
**Pull request by annando**
In a previous PR several parts of the notification system had been replaced with a new functionality. These now unused parts are removed with this PR.

### CODE REVIEW

File: include/enotify.php [hunk 1]
```diff
@@ -118,316 +109,234 @@ function notification($params)
...
-			}
-
-			$dest_str = sprintf($message, $params['source_name'], $item_post_type, $title);
-		// "their post"
-		} elseif ($item['author-link'] == $params['source_link']) {
-			if (!empty($params['activity']['origin_comment'])) {
-				$message = $l10n->t('%1$s replied to you on their %2$s %3$s');
-			} elseif (!empty($params['activity']['explicit_tagged'])) {
-				$message = $l10n->t('%1$s tagged you on their %2$s %3$s');
-			} else {
```
Comment (MrPetovan) [inline include/enotify.php:204]: Why did you remove these distinctions?

Comment (annando) [reply include/enotify.php:204]: This is handled in another part of the system now.

**Review by MrPetovan — APPROVED** [review]
